data-prepper-api/src/test/java/org/opensearch/dataprepper/model/source/SourcePartitionTest.java

```diff
@@ -6,6 +6,8 @@
 package org.opensearch.dataprepper.model.source;
```

Comment:
I think this should be moved to `model/source/coordinator` directory? and change the package name accordingly.

Reply:
Thanks for catching this!